### ISSUE TITLE
Do you have Chinese?

### ISSUE THREAD

**Issue by Hiddensu** [kingToolbox/WindTerm#114]
Do you have Chinese? How to set it?

**Comment by EverythingGoesWithTheFlow** [comment]
问过了 意思是软件本身可以设置语言，但是没有对应的语言包 貌似说今年最后一个季度发布

**Comment by kingToolbox** [comment]
Yes, there is no technical problem with supporting multiple languages, but it requires a lot of physical work. There are still many features that need to be developed, so it is expected that the implementation of multiple languages will not be done until the fourth quarter. 

Please keep attation and thank you.

**Comment by kingToolbox** [comment]
The new [WindTerm_2.2.0_Prerelease_4](https://github.com/kingToolbox/WindTerm/releases/tag/2.2-prerelease) version has been released and the Chinese interface has been supported, please download and check it, thank you.
